l10nfr.properties

```diff
@@ -21,7 +21,13 @@ defaults.mavenArtifactId=sonar-l10n-fr-plugin
 9.8.0.changelogUrl=https://github.com/jycr/sonar-l10n-fr/releases/tag/9.8.0
 
 9.8.1.description=Fix misleading translations and epicene writing
-9.8.1.sqVersions=[9.8,LATEST]
+9.8.1.sqVersions=[9.8,9.8]
 9.8.1.date=2023-01-16
 9.8.1.downloadUrl=https://github.com/jycr/sonar-l10n-fr/releases/download/9.8.1/sonar-l10n-fr-plugin-9.8.1.jar
 9.8.1.changelogUrl=https://github.com/jycr/sonar-l10n-fr/releases/tag/9.8.1
+
+9.9.0.description=Support SonarQube 9.9+
+9.9.0.sqVersions=[9.8,LATEST]
```

Comment:
Shouldn't this start at 9.9?

Reply:
I left in the properties file the localizations for 9.8. This version is therefore also compatible with SonarQube 9.8.

Reply:
Since you're leaving 9.8.1 public then this needs to start at 9.9. Otherwise the Marketplace doesn't know which plugin version to offer SQ 9.8 users

Reply:
Thanks for these valuable tips.

If you have any detailed documentation on all these subtleties, I'm interested :)